draft-ietf-mmusic-ice-sip-sdp-36.xml

```diff
@@ -289,7 +289,11 @@ a=candidate:2 1 UDP 1694498815 192.0.2.3 45664 typ srflx raddr
 When an offerer generates the initial offer, in each "m=" section it MUST
 include SDP candidate attributes for each available candidate associated
 with the "m=" section. In addition, the offerer MUST include an SDP ice-ufrag
-and an SDP ice-pwd attribute in the offer.
+attribute, an SDP ice-pwd attribute and an SDP ice-options attribute with
+an "ice2" attribute value the offer. If the offerer is a full ICE implementation,
```

Comment:
an "ice2" attribute value *in* the offer

Reply:
I will fix that.